src/Engine/Component/GeometryComponent.hpp

```diff
@@ -175,7 +175,7 @@ class RA_ENGINE_API PointCloudComponent : public GeometryComponent
     // directly hold a reference to the displayMesh to simplify accesses in handlers
     std::shared_ptr<PointCloud> m_displayMesh {nullptr};
     // The diameter of the splat when rendered
-    float m_splatSize {0.0025};
+    float m_splatSize {0.0025f};
```

Comment:
Why not using Scalar ?

Reply:
Well, I think most of the OpenGL related stuff is float ... but I just change the suffix to remove warning (error ?) from MSVC ...

Reply:
And since I do not know more about this class, I prefer to modify as little as possible.

Reply:
Fine !